Schools/_posts/2018-09-25-AEPSHEP.md

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
So these `_posts` entries do generate webpages that can be accessed from the navigation through all of the posts that HSF makes (meetings, announcements, etc). 

e.g., this one is accessible at <http://127.0.0.1:4000/schools/2018/09/25/AEPSHEP.html> in a local Jekyll environment.

So you need to make sure they have a link to the original school (and it's better to just have a link than be repetitive of information that's elsewhere).

Reply:
Is this what you meant? 

```suggestion
---
title: AEPSHEP (Asia-Europe-Pacific School of High-Energy Physics)
source: http://2018.aepshep.org/
date: 2018-09-25
author: Guillermo Fidalgo
---
[AEPSHEP](http://2018.aepshep.org/)

```